native/src/modules/common/components/ProgressSpinner.tsx

```diff
@@ -38,17 +38,16 @@ export type PropsType = {
 class ProgressSpinner extends React.Component<PropsType> {
   render() {
     const { t, progress, theme } = this.props
+    const imageProps = { 
+      width: logoSize,
+      height: logoSize,
+      transform: `translate(${logoXY}, ${logoXY})`,
+      xlinkHref: buildConfigAssets().loadingImage
+    }
```

Comment:
Isn't this basically just avoiding the issue? Does transform do anything?

Reply:
I don't even get why ts does not throw an error anyway tbh :D

Reply:
Yes, it's only a workaround. I tried to add
declare module '*.svg' {
  import { SvgProps } from "react-native-svg";
  const content: React.FC<SvgProps>;
  export default content;
}
to assets.d.ts to have the types, but that didn't help.

I wouldn't replace transform because as I saw it is a standard svg action.

But I will have another look how we can convince typescript that transform exiists..

Reply:
And yes, it only does a little bit of centering, perhaps we could also find another way

Reply:
It does seem like it does not exist as prop: https://github.com/react-native-svg/react-native-svg/blob/develop/src/elements/Image.tsx. Maybe you can just use custom styling with a StyledComponent?

Reply:
Easy and clean solution: Added a wrapper for transform